go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/stratoberry/go-gpsd v1.1.0
 	github.com/stretchr/testify v1.8.0
 	golang.org/x/sync v0.1.0
+	k8s.io/api v0.25.4
```

Comment:
Is this change necessary?

Reply:
Looks like client-go requires that:
k8s.io/client-go@v0.25.4 k8s.io/api@v0.25.4
This appeared after I added unit tests that use fake client